src/core/service.c

```diff
@@ -947,6 +947,19 @@ static void service_search_main_pid(Service *s) {
                 log_unit_warning_errno(UNIT(s), r, "Failed to watch PID "PID_FMT" from: %m", pid);
 }
 
+bool service_exited_release_console(Unit *u){
+        Service *s;
+
+        s = SERVICE(u);
+        if (!s)
+            return false;
+        /* For remain_after_exit services, let's see if we can "release" the
```

Comment:
This comment needs to be updated. This function is now called from `unit_notify`, so "unit_notify() only does that in case of change of state" is not true anymore.